vector/src/main/java/im/vector/view/ActiveWidgetsBanner.java

```diff
@@ -219,9 +222,19 @@ public void onActivityResume() {
      * The parent activity is suspended
      */
     public void onActivityPause() {
-        WidgetsManager wm = WidgetManagerProvider.INSTANCE.getWidgetManager(getContext());
+        WidgetsManager wm = getWidgetManager(getContext());
         if (wm != null) {
             wm.removeListener(mWidgetListener);
         }
     }
+
+    @Nullable
+    private WidgetsManager getWidgetManager(Context activity) {
```

Comment:
This code is duplicated with the one in VectorOngoingConferenceCallView. Can we avoid that

Reply:
Thx, forgot to refactor this one